src/styles/card.scss

```diff
@@ -0,0 +1,79 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  transition-duration: 300ms;
+  transition-property: background-color, color, border;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__info {
+    font-family: $font;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use `font` shorthand property when setting `font-family`, `font-size`, `line-height`, and `font-weight`. For example: `font: 500 12px/18px $font;`.